### DISCUSSION TITLE
How to make donations?

### DISCUSSION THREAD

**Discussion by kristofmulier** [marcelotduarte/cx_Freeze#938] [Ideas]
Hi @marcelotduarte ,
We're using your software `cx_Freeze` to build our python application into an executable for both Windows and Linux (see https://embeetle.com). I would like to make a small donation to express my gratitude for your work.

How do I do that? I cannot find any "donate" button here.

Kind regards,
Kristof Mulier

**Comment by marcelotduarte** [comment]
I enabled the sponsor button via PayPal for now. Thanks!
cx_Freeze 6.6 has just been released.
